sdk/monitor/azure-monitor-opentelemetry-autoconfigure/src/main/java/com/azure/monitor/opentelemetry/autoconfigure/AzureMonitorAutoconfigure.java

```diff
@@ -18,9 +18,9 @@
 /**
  * Class to enable Azure Monitor for OpenTelemetry autoconfiguration.
  */
-public final class AzureMonitorExporter {
+public final class AzureMonitorAutoconfigure {
```

Comment:
let's follow OpenTelemetry's casing:

```suggestion
public final class AzureMonitorAutoConfigure {
```